src/main/java/io/odpf/depot/bigquery/client/BQTableDefinition.java

```diff
@@ -1,64 +0,0 @@
-package io.odpf.depot.bigquery.client;
-
-import com.google.cloud.bigquery.Field;
-import com.google.cloud.bigquery.LegacySQLTypeName;
-import com.google.cloud.bigquery.Schema;
-import com.google.cloud.bigquery.StandardTableDefinition;
-import com.google.cloud.bigquery.TimePartitioning;
-import io.odpf.depot.bigquery.exception.BQPartitionKeyNotSpecified;
-import io.odpf.depot.config.BigQuerySinkConfig;
-import lombok.AllArgsConstructor;
-
-import java.util.Optional;
-
-@AllArgsConstructor
-public class BQTableDefinition {
```

Comment:
Keep this class. and add new functions here. 
let's not change the name of the class.